File: BungiiAutomation/Bungii.Test.Regression.Android.Integration/Pages/MenuPages/FAQPage.cs
using OpenQA.Selenium;
using OpenQA.Selenium.Support.PageObjects;

namespace Bungii.Test.Regression.Android.Integration.Pages
{
    class FAQPage
    {
        public FAQPage(IWebDriver driver)
        {
           PageFactory.InitElements(driver, this);
        }
        //------Title-------------------------------------------------------------------------------
        [FindsBy(How = How.XPath, Using = "//android.widget.TextView[@text='FAQ']")]
        public IWebElement Header_FAQPage { get; set; }

        //------Page Elements------------------------------------------------------------------------
        [FindsBy(How = How.Id, Using = "com.bungii.customer:id/promo_code_label")]
        public IWebElement FAQ_BungiiLogo { get; set; }

        [FindsBy(How = How.Id, Using = "")]
        public IWebElement FAQ_Image { get; set; }

        [FindsBy(How = How.Id, Using = "")]
        public IWebElement FAQ_TitleImage { get; set; }

        [FindsBy(How = How.Id, Using = "")]
        public IWebElement FAQ_AppFAQTitle { get; set; }

        [FindsBy(How = How.XPath, Using = "//android.view.View[@index='2' and @content-desc='I have a question. + ' and @instance='20']")]
        public IWebElement FAQ_FirstQuestion { get; set; }

        [FindsBy(How = How.XPath, Using = "//android.view.View[@content-desc='We are more than happy to help. Please text/call customer support at (913) 353-6683.' and @instance='23']")]
        public IWebElement FAQ_FirstAnswer { get; set; }

        [FindsBy(How = How.Id, Using = "-h2-data-preserve-html-node-true-style-font-size-20px-what-if-there-s-an-issue-during-my-bungii-trip-h-data-preserve-html-node-true-style-float-right-h-hr-data-preserve-html-node-true-style-width-100-h2-")]
        public IWebElement FAQ_LastQuestion { get; set; }
        
        [FindsBy(How = How.XPath, Using = "//android.view.View[@content-desc='Please contact text support at (913) 353-6683 or email us at support@bungii.com immediately.' and @instance='54']")]
        public IWebElement FAQ_LastAnswer { get; set; }

        [FindsBy(How = How.XPath, Using = "//android.webkit.WebView[@content-desc='App FAQ']/android.view.View[4]/android.view.View[5]/android.view.View/android.view.View[@content-desc='bungiiapp'][1]")]
        public IWebElement FAQ_TwitterLogo { get; set; }

        [FindsBy(How = How.XPath, Using = "//android.webkit.WebView[@content-desc='App FAQ']/android.view.View[4]/android.view.View[5]/android.view.View/android.view.View[@content-desc='bungiiapp'][2]")]
        public IWebElement FAQ_InstagramLogo { get; set; }

        [FindsBy(How = How.XPath, Using = "//android.webkit.WebView[@content-desc='App FAQ']/android.view.View[4]/android.view.View[5]/android.view.View/android.view.View[@content-desc='bungiiapp'][3]")]
        public IWebElement FAQ_FBLogo { get; set; }
    }
}

File: Bungii.Test.Integration/src/test/java/com/bungii/android/pages/menus/FAQPage.java
package com.bungii.android.pages.menus;

import com.bungii.common.core.PageBase;
import org.openqa.selenium.WebElement;

public class FAQPage extends PageBase {

    //------Title-------------------------------------------------------------------------------
    public WebElement Header_FAQPage() { return findElement("//android.widget.TextView[@text='FAQ']", LocatorType.XPath); }

    //------Page Elements------------------------------------------------------------------------
    public WebElement FAQ_BungiiLogo() { return findElement("com.bungii.customer:id/promo_code_label", LocatorType.Id); }

    public WebElement FAQ_Image() { return findElement("", LocatorType.Id); }

    public WebElement FAQ_TitleImage() { return findElement("", LocatorType.Id); }

    public WebElement FAQ_AppFAQTitle() { return findElement("", LocatorType.Id); }

    public WebElement FAQ_FirstQuestion() { return findElement("//android.view.View[@index='2' and @content-desc='I have a question. + ' and @instance='20']", LocatorType.XPath); }

    public WebElement FAQ_FirstAnswer() { return findElement("//android.view.View[@content-desc='We are more than happy to help. Please text/call customer support at (913) 353-6683.' and @instance='23']", LocatorType.XPath); }

    public WebElement FAQ_LastQuestion() { return findElement("-h2-data-preserve-html-node-true-style-font-size-20px-what-if-there-s-an-issue-during-my-bungii-trip-h-data-preserve-html-node-true-style-float-right-h-hr-data-preserve-html-node-true-style-width-100-h2-", LocatorType.Id); }

    public WebElement FAQ_LastAnswer() { return findElement("//android.view.View[@content-desc='Please contact text support at (913) 353-6683 or email us at support@bungii.com immediately.' and @instance='54']", LocatorType.XPath); }

    public WebElement FAQ_TwitterLogo() { return findElement("//android.webkit.WebView[@content-desc='App FAQ']/android.view.View[4]/android.view.View[5]/android.view.View/android.view.View[@content-desc='bungiiapp'][1]", LocatorType.XPath); }

    public WebElement FAQ_InstagramLogo () { return findElement("//android.webkit.WebView[@content-desc='App FAQ']/android.view.View[4]/android.view.View[5]/android.view.View/android.view.View[@content-desc='bungiiapp'][2]", LocatorType.XPath); }

    public WebElement FAQ_FBLogo () { return findElement("//android.webkit.WebView[@content-desc='App FAQ']/android.view.View[4]/android.view.View[5]/android.view.View/android.view.View[@content-desc='bungiiapp'][3]", LocatorType.XPath); }
}
